client/client.go

```diff
@@ -291,7 +291,7 @@ func (cli *Client) getAPIPath(ctx context.Context, p string, query url.Values) s
 }
 
 // ClientVersion returns the API version used by this client.
-func (cli *Client) ClientVersion() string {
+func (cli *Client) ClientVersion(_ context.Context) string {
 	return cli.version
 }
```

Comment:
Doesn't look like we're using it here, so not sure we should add

Reply:
I _think_ it might be necessary if we want full trace visibility of the call stack from otel. But I could be wrong here and maybe it's unnecessary as you say